src/middlewares/ValidateUser.js

```diff
@@ -0,0 +1,35 @@
+import Joi from '@hapi/joi';
+/**
+ * @export
+ * @class Validations
+ */
+class Validations {
+  /**
+    * User can be able to sign in
+    * @static
+    * @param {object} req request object
+    * @param {object} res response object
+    * @param {object} next next
+    * @memberof Validations
+    * @returns {object} data
+    */
+  static validateSignin(req, res, next) {
+    try {
+      const schema = Joi.object({
+        email: Joi.string().regex(/^\w+@[a-zA-Z_]+?\.[a-zA-Z]{2,3}$/).required(),
+        password: Joi.string().required()
+      });
+      const { error } = schema.validate(req.body);
+
+      if (error) {
+        throw new Error(error.details[0].message);
```

Comment:
Please use Express validator as we agreed on so you will be able to present the user with all errors at once.

Reply:
I am now using `express-validator`.